components/WorkflowStartResponse.yml

```diff
@@ -0,0 +1,48 @@
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC HTTP API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+components:
+  schemas:
+    WorkflowStartResponse:
+      type: object
+      description: A response to a request to start a workflow.
+      properties:
+        verifiablePresentationRequest:
+          $ref: "./VerifiablePresentationRequest.yml#/components/schemas/VerifiablePresentationRequest"
```

Comment:
it is possible to get better refs, with some effort.

Reply:
Fixed in e9dc263eb99297529f56ac4aee950a2211fdf9ba.